_persistence/exec/readme.md

```diff
@@ -5,8 +5,8 @@ title: Exec - Persistence
 type: persistence
 description: "This service allows you to execute commands in the underlying OS to persist item states. It could be used to, for example, call the original `rrdtool` CLI tool."
 source: https://github.com/openhab/openhab1-addons/blob/master/bundles/persistence/org.openhab.persistence.exec/README.md
-since: 1x
-install: manual
+since: 2x
+install: auto
```

Comment:
lines 8 and 9 are due to a bug introduced by this PR which stems from all the synonyms (transform, transformation, transformations) in play.  I'm leaving it as is due to lack of time (but of course feel free to reject the PR for this or any reason).